packages/ui/src/theme/tokens/components/toggleButton.ts

```diff
@@ -1,4 +1,16 @@
-export const togglebutton = {
+export interface ToggleButtonTokens {
+  borderColor: any;
+  color: any;
+  _hover: any;
```

Comment:
Just curious, why is an underscore prepended to the ones related to state?

Reply:
This is a convention we are using to indicate that these are tokens related to [pseudo classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes). We can't use `:` in a property name, or we would be using `:hover`  instead.